### DISCUSSION TITLE
Immer wieder Neg. Rate seit Wochen

### DISCUSSION THREAD

**Discussion by AdmiralRaccoon** [jomjol/AI-on-the-edge-device#2912] [General]
Moin, 
ich hab den AIOTE jetzt seit Oktober 22 an meiner Wasseruhr am laufen, also nichts bewegt oder an der config verstellt (Ausser updates eingespielt, aktuell die 15.6.0).

Seit Anfang des Jahres bekomm ich häufiger (mehrmals die Woche) einen Neg. Rate Error, wobei ich mich da frage, was da falsch läuft.  Ich hatte dann zum testen mal ein neues Reference Image gemacht und die ROIS neu angelegt, aber das lief dann auch nur ein paar Tage gut.

Irgendeiner Idee dazu, bzw. müsste ich noch weitere Infos bereitstellen?
Danke schonmal!
Philipp

So siehts aus:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/41727850/902bc046-d7bd-4a11-839a-8178e958142c)

und das ist die ROI config:
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/41727850/a31cbdee-dc6e-4162-b6dc-9364ee9a918e)

**Comment by caco3** [comment]
We just released [v15.7.0](https://github.com/jomjol/AI-on-the-edge-device/releases/tag/v15.7.0) which hopefully fixes this issue.
